CONTRIBUTING.md

```
@@ -1,9 +1,9 @@
 # Contributing
 
 To keep all docs in one place, we've moved the contributing guide here:
-https://docs.onlook.com/contributing/developers
+https://docs.onlook.com/developers
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)